README.md

```diff
@@ -38,6 +38,18 @@ trayer --edge bottom \
 # forget about i3bar (｢・ω・)｢
 ```
 
+# Components Configuration
+
+## github-alert
```

Comment:
As it is README, you could just go with a literal presentation, like "github alert", "GitHub alert" or anything literal.

Joining two words with hyphen may mislead to others think it is related to the code, e.g. module name or variable name.

This is not necessary to change but recommended.